DQM/TrackingMonitor/interface/TrackAnalyzer.h

```diff
@@ -136,6 +136,21 @@ class TrackAnalyzer
 	    , NumberOfValidRecHitsPerTrackVsEta(NULL)
 	    , NumberOfValidRecHitVsPhiVsEtaPerTrack(NULL)
 
+            , NumberOfLostRecHitsPerTrackVsPhi(NULL)
```

Comment:
Correct, but it could reveal more issues (e.g., it did happen in Pythia8, because they kept checking garbage pointers in if-statements).

I see, some members are set to `nullptr` is `TrackAnalyzer::initHistos()`. So there are 3 different places there it's being done.

It seems that `doDCAPlots_` must be true.

```
(gdb) info b
Num     Type           Disp Enb Address            What
1       breakpoint     keep y   0x00007fffbf4dfa0f in TrackAnalyzer::bookHistosForHitProperties(DQMStore::IBooker&) at /data/davidlt/fix_root/CMSSW_7_6_DEVEL_X_2015-09-01-2300/src/DQM/TrackingMonitor/src/TrackAnalyzer.cc:411
```

```
   410          histname = "xPointOfClosestApproachToPV_";
   411          xPointOfClosestApproachToPV = ibooker.book1D(histname+CategoryName, histname+CategoryName, VXBin, VXMin, VXMax);
   412          xPointOfClosestApproachToPV->setAxisTitle("x component of Track PCA to pv (cm)",1);
   413          xPointOfClosestApproachToPV->setAxisTitle("Number of Tracks",2);
```

So, I guess `doDCAPlots_ ` isn't `true`. Breakpoint didn't hit it.